core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -117,7 +117,7 @@ angular.module('oppia').directive('audioTranslationBar', [
         $('.oppia-translation-tab').on('drop', function(evt) {
           evt.preventDefault();
           if (
-            (evt.target as Element).classList.contains(
+            (evt.target as unknown as Element).classList.contains(
```

Comment:
Why `unknown`?

Reply:
The jQueries target event to Element conversion is not possible according to ts compiler.

Reply:
After jQuery is removed this won't be a problem anymore, right?

Reply:
Yes